### DISCUSSION TITLE
Angular prod build would bundle json files ( used by tailwind only )

### DISCUSSION THREAD

**Discussion by ivanchenhz** [tailwindlabs/tailwindcss#8376] [Help] [answered]
**What version of Tailwind CSS are you using?**

For example: v3.0.1

**What build tool (or framework if it abstracts the build tool) are you using?**

For example: Angular 11.2.0, 

**What version of Node.js are you using?**

For example: v14.17.5

**What browser are you using?**

For example: Chrome, Safari, or N/A

**What operating system are you using?**

For example: macOS

**Reproduction URL**

I attached snapshots, hope it's enough for describing the issue.

**Describe your issue**
We used some json stuff to help generate the classes for color, icon, like:
```json
 "theme-common-text-white": "var(--theme-common-text-white)",
  "theme-common-text-black": "var(--theme-common-text-black)",
```

It works perfect, except we noticed the prod build actually bundle the json files ( we are unnecessary, not sure why ) 

<img width="308" alt="Screen Shot 2022-05-18 at 9 37 29 AM" src="https://user-images.githubusercontent.com/2640513/168940190-63df2a2c-6ef7-411e-95c9-69aac0b5d9c8.png">
<img width="683" alt="Screen Shot 2022-05-18 at 9 39 12 AM" src="https://user-images.githubusercontent.com/2640513/168940198-c3f794a0-0abc-4bba-bc26-3e2589a5748a.png">

**Comment by adamwathan** [comment, marked as answer]
Hey! Need a reproduction here I’m afraid. Are you requiring your tailwind.config.js file in your frontend anywhere? That’s the only reason I can imagine it would be bundled.

**Reply by ivanchenhz** [reply]
I don't think we require the config file, will setup a repo for reproducing,

**Reply by ivanchenhz** [reply]
Hey, thanks Adam,
Turns out you are right, we do import the config file ..., ( over half an year ago, for breakpoint change notification ... 😂 ）

```js
src/app/utils/breakpoints.ts:
   3: import * as tailwind from '../../../tailwind.config';
  52:   const screens = tailwind.theme.screens;
```

**Reply by ivanchenhz** [reply]
By the way, when setting the reproduction, I noticed one 'known issue' still there, so report a new issue: https://github.com/tailwindlabs/tailwindcss/issues/8382

Again, thanks for pointing out the 'requiring config', we didn't think it this way 😄 👍
